Packs/CrowdStrikeFalconSandbox/Integrations/integration-CrowdStrikeFalconSandbox.yml

```diff
@@ -606,11 +622,20 @@ script:
         throw ('Timeout due to no answer after ' + timeOut + ' seconds.');
     }
 
-    function submitUrlCmd(url, environmentID) {
+    function submitUrlCmd(url, environmentID, dontThrowErrorOnFileDetonation) {
         if (version === 'v1') {
             throw 'This command is supported only in API v2.'
         }
-        var response = submitUrl(url, environmentID);
+        try {
+          var response = submitUrl(url, environmentID);
+        } catch (exception) {
+              var notSupported = 'The provided URL resolves to a file.'
+              if (dontThrowErrorOnFileDetonation && exception.contains(notSupported)){
+                  return notSupported;
```

Comment:
lets return a better error message here that the file type is not supported

Reply:
It does not support any files. Just webpages.

Reply:
change to something like `The file format is not supported, use <COMMAND NAME> instead.`